test/source/tests/decrypt.ts

```diff
@@ -25,6 +25,23 @@ export const defineDecryptTests = (testVariant: TestVariant, testWithBrowser: Te
 
   if (testVariant !== 'CONSUMER-LIVE-GMAIL') {
 
+    ava.default(`decrypt - outlook message with ATTxxxx encrypted email doesn't show empty attachment`, testWithBrowser('compatibility', async (t, browser) => {
+      const threadId = '17dbdf2425ac0f29';
+      const acctEmail = 'flowcrypt.compatibility@gmail.com';
+      const inboxPage = await browser.newPage(t, TestUrls.extension(`chrome/settings/inbox/inbox.htm?acctEmail=${acctEmail}&threadId=${threadId}`));
+      expect(await inboxPage.isElementPresent('@attachments')).to.equal(false);
```

Comment:
well, it's not that simple. We should await for some readiness indication (please see what's inside InboxPageRecipe.checkDecryptMsg, something like     `await inboxPage.waitAll('iframe');`).
Need to double-check that this test actually fails if we don't mark ATT00001 as 'hidden'

Reply:
sir @rrrooommmaaa, I have added a waiting time for the webpage to complete and also wait for the iframe before testing the element.

>Need to double-check that this test actually fails if we don't mark ATT00001 as 'hidden'

I have checked it manually by reverting all changes on the `mime.ts` and the following tests above failed when reading the message through the FlowCrypt inbox.

Reply:
That's good. However using `sleep` is not reliable and should be used as last resort only.
Have you considered waiting for 'ready' test state, as implemented in many elements and their tests?
For example, we have `Ui.setTestState('ready');` as the last statement in `SettingsView.render()` in 
`extension/chrome/settings/index.ts`.
Would it help if we implement the same mechanism for `InboxView` too?

Reply:
Thank you, sir! I have used the `waitForSelTestState('ready')` in replacement for `sleep()` when waiting for the body element's ready state.